mail_suggested_recipient_unchecked/models/mail_thread.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _message_add_suggested_recipient(
+        self, result, partner=None, email=None, lang=None, reason=""
+    ):
+        """Uncheck all the suggested recipients updating the dictionary value."""
```

Comment:
```suggestion
        # Uncheck all the suggested recipients updating the dictionary value.
```
Add this as a comment, otherwise you're overwriting the original method's docstring.